src/components/OpenSourceBlurb.astro

```diff
@@ -0,0 +1,73 @@
+---
+import { Image } from "astro:assets";
+import gradientLineImg from "../images/gradient-line.svg";
+import discordIllustrationImg from "../images/discord-illustration.svg";
+import branchingImg from "../images/branching.svg";
+import heartImg from "../images/heart.svg";
+import Card from "./Card.astro";
+---
+
+<section class="container mx-auto pt-16 lg:py-32 px-4">
+  <div class="max-w-fit mx-auto mb-5 lg:mb-16 flex flex-col items-center gap-2">
+    <Image src={gradientLineImg} alt="A gradient line." class="mb-2" />
+    <h1 class="font-bold text-3xl">
+      Grain
+      <Image src={heartImg} alt="A heart." class="inline" />
+      open source
+    </h1>
+    <p>Grain is a passion project. All of our projects are <a href="https://github.com/grain-lang" class="text-color-accent underline hover:opacity-80">open source</a>.</p>
+  </div>
+  <div class="mx-auto">
+    <div class="grid grid-cols-1 lg:grid-cols-2 gap-5 lg:gap-14">
+      <Card title="We continuously strive to" class="bg-color-background-bright">
+        <ul class="pl-6 [&>li]:my-1 [&>li]:pl-1 text-color-accent list-outside list-image-[url(/src/images/check-icon.svg)]">
+          <li>Improve the language</li>
+          <li>Address more use cases and provide exceptional documentation</li>
+          <li>Prioritize developer experience</li>
+          <li>Facilitate an active, engaged, and respectful community</li>
+        </ul>
+        <p>
+          <b>Our bottom line:</b>
+          Help you reap the benefits of Grain and WebAssembly, with an
+          effortless language, great developer experience, broad application
+          potential, and an awesome community.
+        </p>
+      </Card>
+      <Card title="Sponsors and Contributors" class="bg-color-background-bright relative">
+        <div class="z-10 absolute -top-10 right-[10rem] bg-color-blurb-4 w-36 h-[32rem] rounded-full rotate-[10deg] blur-3xl" data-desc="color-ellipsis" />
+        <div class="z-10 absolute -top-10 right-[3rem] bg-color-blurb-2 w-36 h-[32rem] rounded-full rotate-[10deg] blur-3xl" />
+        <div class="z-10 absolute top-0 -right-[3rem] bg-color-blurb-3 w-36 h-[32rem] rounded-full rotate-[10deg] blur-3xl" data-desc="color-ellipsis" />
+        <p class="z-20">
```

Comment:
I found this hard to read because of the gradient in the background, I think if we made this background black with a 50% opacity and some very slight background blur it would greatly improve readability while still keeping the nice gradient.